src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1546,7 +1554,7 @@ public Worker build() {
                     workerStateChangeListener,
                     shardSyncer,
                     leaderDecider,
-                    null /* PeriodicShardSyncManager */);
```

Comment:
why are we passing this now? will this change any behavior?

Reply:
That needs to be passed now because the StreamsWorkerFactory now passes the DynamoDBPeriodicShardSyncManager to the Worker, so the periodicShardSyncManager object now needs to be passed to the Worker. It shouldn't change behavior because if a PSSM is not passed, the object will be null and the behavior will be as normal with the getOrCreatePeriodicShardSyncManager function creating a KinesisPeriodicShardSyncManager